packages/cyberstorm/src/components/CopyButton/CopyButton.module.css

```diff
@@ -0,0 +1,11 @@
+.copy {
+  color: var(--color-text--tertiary);
+}
+
+.checkmark {
+  color: var(--color-cyber-green--50);
+}
+
+.copy:hover {
```

Comment:
It would be clearer if this would come after `.copy` instead of `.checkmark`, to have `.copy`-related stuff close to each other.